### PR TITLE
Use iframe-resizer

### PR DESCRIPTION
**Pull request by rschick**
I've tested this with Chrome, Firefox and Safari on Mac, and iframe-resizer works better than the simple scrollHeight change we were using before: the iframe will shrink as well as grow, whereas before it wouldn't shrink.

I had to modify the iframe-resizer library to get it to load for tests. I submitted a PR and will update here when that gets accepted.

### CODE REVIEW

**Comment by dbatiste** [comment]
A couple more things we'll need to do:
* test for IE9+
* update monolith 3rd-party tool references to include reference to this library

**Comment by dbatiste** [comment]
Does this require the client page to include "iframeResizer.contentWindow.min.js" on their page?

**Comment by rschick** [comment]
No, it's bundled in with ifrau. So the client page just needs to include ifrau.

**Comment by rschick** [comment]
By "test for IE9+" do you mean do QA testing, or test for IE9+ in code and take some action?

I've never updated the monolith 3rd-party tool references - could you point me to where that is?

**Comment by dbatiste** [comment]
I thinking some QA testing to make sure it works to our satisfaction there.  Just skimming the docs, it sounds like it can calculate the dimensions in different ways, and may or may not rely on MutationObserver depending on browser support.  It sounds like it should work, hopefully there aren't any flickery effects in either case.

For sure - I will send you an email with some details about the 3rd-party references.

File: test/host.js [hunk 1]
```diff
@@ -87,13 +92,25 @@ describe('host', () => {
...
+			host.receiveEvent('title', ['new title']);
+			global.document.title.should.equal('new title');
+		});
+
+		it('should update the document location', () => {
+			host.connect();
+			host.receiveEvent('navigate', ['new url']);
+			global.document.location.href.should.equal('new url');
+		});
+
```
Comment (dlockhart) [inline test/host.js]: Thanks for adding these bonus tests :)

**Comment by dlockhart** [comment]
This looks good to me. Once you've confirmed everything is working in IE9 (which I'm sure it will), let me know and I'll merge and do another release.

**Comment by rschick** [comment]
For testing with IE, could I publish this branch to npm?

**Pull request merged**